doc/user_guide/configuration/security.rst

```diff
@@ -50,17 +74,138 @@ Specific examples
                        , password='password')
 
 
-2) How to connect with TLS encryption and fingerprint verification:
+#. Connect to Exasol SAAS (TLS encryption is REQUIRED for SAAS):
 
    .. code-block:: python
 
-      pyexasol.connect(dsn='myexasol/135a1d2dce102de866f58267521f4232153545a075dc85f8f7596f57e588a181:8563'
+      pyexasol.connect(dsn='abc.cloud.exasol.com:8563'
                        , user='user'
-                       , password='password'
+                       , refresh_token='token'
                        )
 
+   .. code-block:: python
+
+      pyexasol.connect(dsn='myexasol:8563'
+                       , user='user'
+                       , access_token='personal_access_token'
+                       )
+
+
+
+Transport Layer Security (TLS)
+******************************
+
+Similar to other Exasol connectors, PyExasol is compatible with using TLS cryptographic
+protocol. As a part of the TLS handshake, the drivers require the SSL/TLS certificate
+used by Exasol to be validated. This is the standard practice that increases the security of
+connections by preventing man-in-the-middle attacks.
+
+Please check out the following documentation for user-friendly tutorials on TLS from Exasol:
+
+* `An introduction to TLS <https://github.com/exasol/tutorials/blob/1.0.0/tls/doc/tls_introduction.md>`__
+* `TLS at Exasol <https://github.com/exasol/tutorials/blob/1.0.0/tls/doc/tls_with_exasol.md>`__
+* `TLS in UDFs tutorial <https://github.com/exasol/tutorials/blob/1.0.0/tls/doc/tls_in_udfs.md>`__
+
+For technical articles made by Exasol relating to TLS, please see:
+
+- `Database connection encryption at Exasol <https://exasol.my.site.com/s/article/Database-connection-encryption-at-Exasol/>`__
+- `CHANGELOG: TLS for all Exasol drivers <https://exasol.my.site.com/s/article/Changelog-content-6507>`__
+- `CHANGELOG: Database accepts only TLS connections <https://exasol.my.site.com/s/article/Changelog-content-16927>`__
+- `Generating TLS files yourself to avoid providing a fingerprint <https://exasol.my.site.com/s/article/Generating-TLS-files-yourself-to-avoid-providing-a-fingerprint/>`__
+- `TLS connection fails <https://exasol.my.site.com/s/article/TLS-connection-fails>`__
+
+
+Fingerprint Verification
```

Comment:
verification or validation? want more information for (docs/links)?
should we make a note for https://github.com/exasol/pyexasol/issues/185 ?